moxin-frontend/src/chat/model_selector.rs

```diff
@@ -234,6 +234,36 @@ impl Widget for ModelSelector {
     }
 
     fn draw_walk(&mut self, cx: &mut Cx2d, scope: &mut Scope, walk: Walk) -> DrawStep {
+        let store = scope.data.get::<Store>().unwrap();
+
+        let mut downloaded_files = store.downloaded_files.clone();
+        downloaded_files.sort_by(|a, b| b.downloaded_at.cmp(&a.downloaded_at));
```

Comment:
No need to sort here, right? 

Also, in this case it seems easy to avoid the clone, by doing:
`let no_available_models = store.downloaded_files.is_empty()`

Reply:
oops, this is what happens when you recklessly copy paste, fixed